src/lib/css-tag.ts

```diff
@@ -42,6 +42,10 @@ export class CSSResult {
     }
     return this._styleSheet;
   }
+
+  toString(): String {
```

Comment:
Shouldn't this be type `string` ?

Reply:
Absolutely. Good catch!

Reply:
https://github.com/Polymer/lit-element/pull/546